docs/reference/concepts/connection-flows.md

```diff
@@ -3,66 +3,53 @@
+* Connecting a server-worker instance to a cloud deployment.
+* (For debugging) Connecting a server-worker or a client-worker instance to a cloud deployment from your local machine.
 
-### Locator connection flow
-From SpatialOS v13.5, there are two versions of the Locator connection. The new v13.5+ Locator, in alpha, has additional functionality to the existing v10.4+ Locator which is the stable version. 
+<%(#Expandable title="How do I connect to a cloud deployment from my local machine?")%>
+There are three steps for connecting a worker to a cloud deployment:
 
+1. In your terminal, execute `spatial cloud connect external <deployment_name>`, where `<deployment_name>` is the deployment you want to connect to.
+2. Ensure that `UseExternalIp` is set to true in the connection parameters for the worker you want to connect.
+3. Run your worker, either in the Unity Editor or built out as if you were connecting to a local deployment.
 
-### v10.4+ Locator connection flow (stable version)
+Note that this debugging flow **does not work** for mobile clients. You need to use the new v13.5+ Locator connection flow instead.
 
-Use this Locator service connection flow for:
+For more information about the `spatial cloud connect external` command, see the [CLI documentation](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-cloud-connect-external).
 
- * Connecting a client-worker to a cloud deployment via the SpatialOS Launcher - [see SpatialOS documentation on the Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher)
+<%(/Expandable)%>
 
+### Locator connection flow
 
-### New v13.5+ Locator connection flow (alpha version)
+From SpatialOS v13.5 onwards, there are two versions of the Locator connection. The new v13.5+ Locator, in alpha, has additional functionality to the existing v10.4+ Locator which is the stable version.
 
-Use this Locator service connection flow for:
+#### v10.4+ Locator connection flow (stable version)
 
-* Connecting a client-worker to a cloud deployment via the SpatialOS Launcher - [see SpatialOS documentation on the Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher)
-* Connecting a client-worker instance to a cloud deployment from your Unity Editor for debugging via the [development authentication functionality](https://docs.improbable.io/reference/13.5/shared/auth/development-authentication). (Note that you can also use the Receptionist to connect in this situation.)
+Use this Locator connection flow for:
 
-## When to use the `Connection` object
+* Connecting a client-worker to a cloud deployment via the [SpatialOS Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher).
 
-Upon successfully connecting to the SpatialOS Runtime, your worker stores a `Connection` object.
-Use this object for:
+#### v13.5+ Locator connection flow (alpha version)
```

Comment:
think we should switch the position of the alpha and the stable version here. The Locator v10.4 is not really used  in the GDK

Reply:
Its used for all non-mobile clients

Reply:
talked offline: switch it